v0/util/errorTypes/abortedError.js

```diff
@@ -0,0 +1,28 @@
+const tags = require("../tags");
+const { DefaultError } = require("./default");
+
+class AbortedError extends DefaultError {
+  constructor(
+    message,
+    statusCode = 400,
+    statTags = {},
```

Comment:
as we discussed, lets not support this as this moment

Reply:
Fixed it.